pkgs/development/libraries/qt-5/modules/qt3d.nix

```diff
@@ -1,7 +1,10 @@
-{ qtModule, qtbase, qtdeclarative }:
+{ lib, stdenv, qtModule, qtbase, qtdeclarative }:
 
-qtModule {
+qtModule ({
   pname = "qt3d";
   qtInputs = [ qtbase qtdeclarative ];
   outputs = [ "out" "dev" "bin" ];
-}
+} // lib.optionalAttrs (stdenv.isDarwin && stdenv.isAarch64) {
```

Comment:
Please just inline this. Merging the attr is not that nice and for bigger files reduces readability quite a lot.

Reply:
Done.